lang/python/python/patches/015-openssl-deprecated.patch

```diff
@@ -0,0 +1,67 @@
+--- a/Modules/_ssl.c
++++ b/Modules/_ssl.c
+@@ -94,13 +94,13 @@ struct py_ssl_library_code {
+ #include "_ssl_data.h"
+ 
+ #if (OPENSSL_VERSION_NUMBER >= 0x10100000L) && !defined(LIBRESSL_VERSION_NUMBER)
+-#  define OPENSSL_VERSION_1_1 1
+-#  define PY_OPENSSL_1_1_API 1
++# define OPENSSL_VERSION_1_1 1
++# define PY_OPENSSL_1_1_API 1
```

Comment:
this part of the patch looks like noise

Reply:
Surprisingly it is not.

#define works
# define works
#  define does not

Reply:
oh, ok
my bad; i learned something; i didn't take a close look on this;